docs/docs/components/swipeable.md

```diff
@@ -11,6 +11,11 @@ import GifGallery from '@site/components/GifGallery'
   <img src={useBaseUrl("gifs/sampleswipeable.gif")} height="120" />
 </GifGallery>
 
+:::caution
+This component is deprecated.
+Please use [the reanimated version](/react-native-gesture-handler/docs/components/reanimated_swipeable).
+:::
```

Comment:
Can you also add deprecation notice to the component itself?

Reply:
I'm adding it as a console warn, are there any other places the notice should be added?
![image](https://github.com/software-mansion/react-native-gesture-handler/assets/74246391/9d14ba3f-df52-4e50-9399-0ad61e5ddee2)

Reply:
Do it in [JSDoc please](https://jsdoc.app/tags-deprecated). We really don't want to show there kind of things as warnings (speaking from experience 😅). Warnings aren't really actionable, there's not much you can do besides stopping using the component which may not be possible immediately.

Reply:
> Do it in [JSDoc please](https://jsdoc.app/tags-deprecated).
Instead or additionally to the warn?

Reply:
Added it alongside the warn.

Reply:
Instead of the warning

Reply:
done